api/v1alpha/framework/testcafe.schema.json

```diff
@@ -131,6 +131,36 @@
             "description": "Disable video recording.",
             "type": "boolean"
           },
+          "filter": {
+            "description": "Permits to filter which tests to execute",
```

Comment:
The description is a bit obtuse. The other descriptions I see you pulled straight from the testcafe docs, maybe do the same here? `Allows you to specify which tests or fixtures to run.`

Also, the existing convention for the descriptions seems to be full sentences, including a period.